### PR TITLE
Readme: Add syntax highlighting, fix typos, other cleanups

### PR DESCRIPTION
**Pull request by jordan-brough**
*Description of changes:*

Adds some things from #68 that were omitted from #81, and also some other improvements.  See individual commits for details.

------
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -93,28 +93,24 @@ You can build RIE into a base image. Download the RIE from GitHub to your local
 3. Install the emulator package and change `ENTRYPOINT` to run the new script by adding the following lines to your Dockerfile:
 
     To use the default x86\_64 architecture
-
```
Comment (bodelsc) [inline README.md:96]: This PR is also changing the formatting, it is inconsistently applied as well. The change needs updated to not affect the spacing.

File: README.md [hunk 2]
```diff
@@ -138,17 +134,18 @@ You install the runtime interface emulator to your local machine. When you run t
     ```
 
 2. Run your Lambda image function using the docker run command.
-
-    ```
-    docker run -d -v ~/.aws-lambda-rie:/aws-lambda -p 9000:8080 myfunction:latest
-        --entrypoint /aws-lambda/aws-lambda-rie  <image entrypoint> <(optional) image command>`
```
[the remark this answers is not in the document]
Comment (jordan-brough) [reply README.md:144]: Sure, if you prefer to have the changes broken out as separate commits I'm happy to accommodate.

I've force-pushed breaking out several commits, and also addressed a couple other things in the Readme as well.

Just checking: You did mean separate commits and not separate PRs, right? I'm happy to do separate PRs if you prefer.

File: README.md (hunk 1 above)
Comment (jordan-brough) [reply README.md:96]: Good point. Updated.

Comment (bodelsc) [reply README.md:96]: Thanks!

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (bodelsc) [reply README.md:144]: Sorry, should have clarified I was leaving the nitpick as a comment but did not expect it to be actioned 👍

It seems rebasing and merging is not enabled for the repository. But when that is enabled and done in my opinion It just makes it easier to revert a single thing with a PR than having to revert the entire squashed commit.

File: README.md [hunk 3]
```diff
@@ -121,7 +121,8 @@ You can build RIE into a base image. Download the RIE from GitHub to your local
 ### Test an image without adding RIE to the image
 
 You install the runtime interface emulator to your local machine. When you run the container image, you set the entry point to be the emulator.
-*To test an image without adding RIE to the image *
+
+#### To test an image without adding RIE to the image
```
Comment (bodelsc) [inline README.md:125]: Can this be added to the content table please.

Comment (bodelsc) [reply README.md:125]: Looks good to me bar this comment 👍

**Review by bodelsc — APPROVED** [review]
Look good to me, thank you! 👍 

I will just ask the main maintainer to take a look and merge this.

**Review by valerena — APPROVED** [review]

**Pull request merged**